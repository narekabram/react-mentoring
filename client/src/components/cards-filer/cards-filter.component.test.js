import React from 'react';
import { shallow } from 'enzyme';
import CardsFilter from './cards-filter';

describe('CardsFilter', () => {
    it('Adds CardsFilter snapshot testing', () => {
        const component = shallow(<CardsFilter />);

        expect(component).toMatchSnapshot();
    });
});
