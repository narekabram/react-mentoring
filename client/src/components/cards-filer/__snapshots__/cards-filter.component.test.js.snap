// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardsFilter Adds CardsFilter snapshot testing 1`] = `ShallowWrapper {}`;
